pyflwdir/subgrid.py

```diff
@@ -93,6 +93,55 @@ def segment_area(
     return ucatch_map, ucatch_are
 
 
+@njit
+def ucat_volume(
+    idxs_out,
+    idxs_ds,
+    seq,
+    hand,
+    area,
+    depths=np.arange(0.5, 3.0, 0.5, dtype=np.float32),
```

Comment:
Why not have as default a depth profile interval of 0.5 m?

Reply:
I think that's the case, isn't it?

Reply:
Yes, it is indeed the case!